src-built-in/components/toolbar/components/Search.jsx

```diff
@@ -136,8 +136,15 @@ export default class Search extends React.Component {
 	setActive(err, data) {
 		this.setState({ active: data.value })
 	}
-	hotKeyActive() {
-		this.setState({ active: true, hotkeySet: true })
+	hotKeyActive(...args) {
+		// the first argument seems to be empty but the second value returns
```

Comment:
The first argument is likely an error object, and will be null or undefined if there isn't an error. If there is an error, it should be logged.